src/Makefile.qt.include

```diff
@@ -314,8 +314,7 @@ BITCOIN_QT_CPP += \
   qt/walletview.cpp
 endif
 
-RES_IMAGES = \
-  qt/res/images/about.png
+RES_IMAGES = 
```

Comment:
I suggest removing it since you now set it to empty; so if someone adds another RES_IMAGES to the top of the file you end up deleting it again with this one.